src/App.jsx

```diff
@@ -1,5 +1,21 @@
+import { useState } from "react";
+import { ThemeProvider } from "styled-components";
+import HomeScreen from "./Home/HomeScreen";
+import { lightTheme, darkTheme, GlobalStyles } from "./Common/Theme";
+
 const App = () => {
-  return <div>Hello there!</div>;
-}
+  const [theme, setTheme] = useState("light");
+
+  const themeToggleHandler = () => {
+    theme === "light" ? setTheme("dark") : setTheme("light");
+  };
+
+  return (
+    <ThemeProvider theme={theme === "light" ? lightTheme : darkTheme}>
+      <GlobalStyles />
+      <HomeScreen onThemeToggle={themeToggleHandler} theme={theme} />
```

Comment:
```suggestion
      <HomeScreen onThemeToggle={toggleTheme} theme={theme} />
```
can we use this convention?
@Tanvir04khan